libpod/oci.go

```diff
@@ -49,16 +49,17 @@ const (
 // OCIRuntime represents an OCI-compatible runtime that libpod can call into
 // to perform container operations
 type OCIRuntime struct {
-	name          string
-	path          string
-	conmonPath    string
-	conmonEnv     []string
-	cgroupManager string
-	tmpDir        string
-	exitsDir      string
-	socketsDir    string
-	logSizeMax    int64
-	noPivot       bool
+	name           string
+	path           string
+	conmonPath     string
+	conmonEnv      []string
+	cgroupManager  string
+	tmpDir         string
+	exitsDir       string
+	socketsDir     string
+	logSizeMax     int64
+	noPivot        bool
+	conmonPidFile  string
```

Comment:
Make conmonPidFile part of ContainerConfig, instead of runtime/OCIRuntime - better support for multiple containers.